src/tools/build.sh

```diff
@@ -301,7 +301,7 @@ strip_and_compress $(for i in $SRCLIST; do echo ${i}${EXT}; done)
 ###########################################################################
 
 if [[ -n "$INSTALL" ]]; then
-      cd "${XTLD}/bin/x/$MACOSX_MAJOR_VER"
```

Comment:
Thanks for finding this typo

Reply:
No problem I'm happy to help on such free projects